extensions/shared/src/main/java/app/revanced/extension/shared/patches/spoof/SpoofStreamingDataPatch.java

```diff
@@ -106,6 +93,8 @@ public static void fetchStreams(String url, Map<String, String> requestHeaders)
     @Nullable
     public static ByteBuffer getStreamingData(String videoId, StreamingDataOuterClass$StreamingData originalStreamingData) {
         if (SPOOF_STREAMING_DATA) {
+            streamingDataMap.putIfAbsent(videoId, originalStreamingData);
```

Comment:
Always put originalStreamingData into HashMap regardless of spoofed client type

It's more ideal to put it into HashMap only when spoofed client type is iOS

Reply:
`putIfAbsent` is only available on Android 7.0+

YT Music also uses `Spoof streaming data` patch

YT Music 6.20.51, which is supported by RVX patch, is available on Android 5.0, so if you don't check SDK version, it will crash on Android 5.0